src/style.css

```diff
@@ -1,3 +1,86 @@
+/* stylelint-disable rule-empty-line-before */
```

Comment:
Avoid disabling stylelint rules without a good reason. If you must disable a rule, provide a comment explaining why.